internal/winapi/winapi.go

```diff
@@ -0,0 +1,5 @@
+/*Package winapi contains various low-level bindings to Windows APIs. It can
```

Comment:
Not a fan of multiline comments. Can we use single line instead?

Reply:
```
// Package winapi contains various low-level bindings to Windows APIs. It can
// be thought of as an extension to golang.org/x/sys/windows.
```

Reply:
I'm not either, I grabbed this from Kathryns PR. I thought you guys arrived at a conclusion for the package doc comment but I guess not haha Will change

Reply:
I'll update it in my PR :p